script/release/update_locales

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Download and commit the latest Transifex translations
+#
+
+# Exit on error or uncommitted changes
+# TODO: check that master matches upstream/master
+set -e
+if [ ! -z "$(git status --porcelain)" ]; then
+  echo "Aborted: git working directory is not clean."
+  exit 1
+fi
+
+echo "\n*** Checking out latest master... ***\n"
+git checkout master
+git pull upstream master
+
+echo "\n*** Downloading latest Transifex translations... ***\n"
+tx pull --force
+git commit -a -m "Update all locales with the latest Transifex translations"
+
+echo "\n*** Pushing to master... ***\n"
+git push upstream master
```

Comment:
Maybe this is just me (as a newb) being too cautious - but I have to say I'm a bit weary of having a script push to master :see_no_evil: 

I usually run git status just to make sure I push only transifex files...

Well, let's see what happens! :smile:
(these really do sound like famous last words)

Reply:
Yes it is a little scary to automate it, but after having done it manually so many times it seemed ok.
I did actually include a `git status` check at the start of this script, which aborts if the working directory is not clean, so hopefully that's all covered.

The script still makes a few assumptions (that you have a remote called `upstream`) but hopefully it's safe to assume.